pkg/cli/cmds/server.go

```diff
@@ -280,6 +284,21 @@ func NewServerCommand(action func(*cli.Context) error) cli.Command {
 				Usage:       "(components) Disable " + version.Program + " default network policy controller",
 				Destination: &ServerConfig.DisableNPC,
 			},
+			cli.BoolFlag{
+				Name:        "disable-api-server",
+				Usage:       "(components) Disable running api server",
```

Comment:
We call it kube-apiserver for `kube-apiserver-args` so we should do the same here.

```suggestion
				Name:        "disable-kube-apiserver",
				Usage:       "(components) Disable running kube-apiserver process",
```

Reply:
yeah but I found that we use disable-scheduler for the scheduler disabling originally and for cloud controller manager, so we will have to convert those as well?